### DISCUSSION TITLE
Feature [GitHub Externship]:

### DISCUSSION THREAD

**Discussion by tharani247** [hoppscotch/hoppscotch#1682] [Q&A]
**Creating a Welcome page for Hoppscotch**
There is no welcome page and it is unable to find the what the website is about if a welcome page with brief of hooppscotch website and its features and what it is used for then the end users will be easily able to know about our hoppscotch website

**Creating a Demo Tour for the website**
Even it is difficult to find the how the features work and what are the modules available in the project ,so demo tour will be useful nd easy to understand for the end users

**Providing an option to create a account**
Instead of directly providing the access to google and github we need to provide an option to create an account and then authenticating with google and github

**Even the features are not working properly**
Even though I am providing the correct url and request name it is showing me the network error and try enabling proxy and when I enable proxy it is showing me the profile page as an end user point of view my suggestion is make beautiful interface is not the ultimate goal considering the user queries and trying to respond accordingly will make the website look great

**Comment by tharani247** [comment]
These are features that I found which are not present in the website

**Comment by liyasthomas** [comment]
Can you submit the proposal in [GitHub Externship category](https://github.com/hoppscotch/hoppscotch/discussions/categories/github-externship) under Discussions section.

Feel free to use [this proposal template](https://github.com/hoppscotch/hoppscotch/discussions/1616).

EDIT: I'm transferring this to Discussions - please edit the topic with the proposal template.
